### DISCUSSION TITLE
Documentation for the built-in filter types

### DISCUSSION THREAD

**Discussion by bendykstra94** [TanStack/table#2859] [Ideas]
As I understand from the [docs](https://react-table.tanstack.com/docs/api/useFilters#table-options)

filterTypes: Object<filterKey: filterType>
- Must be memoized
- Allows overriding or adding additional filter types for columns to use. If a column's filter type isn't found on this object, the column will default to using the **built-in filter types**.

there is no documentation for the built ins such as 'between'. When viewing the code-sandbox for filtering I was confused because I had not specified a 'between' filter in the filterTypes object.

Can we add a small section in the useFilter documentation under the column `filter` options: 
`Defaults to 'text'. Other built in options include 'exactText', 'exactTextCase', 'includes', 'includesAll', 'includesSome', 'includesValue', 'exact', 'equals',  and 'between'. See [here](https://github.com/tannerlinsley/react-table/blob/master/src/filterTypes.js) for implementation details`

**Comment by antoniormrzz** [comment]
Can we get some documentation for this? I'm just guessing stuff at this point. `exact`, `includes` etc

**Reply by antoniormrzz** [reply]
Btw, not trying to be rude, thanks for the library!

**Reply by acailly** [reply]
you can check here: https://github.com/TanStack/react-table/blob/v7/src/filterTypes.js
that's not doc but that is better than guessing 😃
